src/java/org/apache/cassandra/db/commitlog/CommitLogDescriptor.java

```diff
@@ -227,6 +227,26 @@ public int getMessagingVersion()
         }
     }
 
+    private static int currentVersion()
+    {
+        switch(MessagingService.current_version)
+        {
+            case MessagingService.VERSION_30:
+            case MessagingService.VERSION_3014:
```

Comment:
Do I understand correctly this is to account for the 30 -> 3014 version bump, as they both translate to `VERSION_30`?

Also what test is this actually fixing (if any)?